mbuild/tests/test_monolayer.py

```diff
@@ -6,6 +8,28 @@
 
 
 class TestMonolayer(BaseTest):
+    def test_betacristobalite(self):
+        # Normal case
+        surface = Betacristobalite()
+        assert surface.n_particles == 1900
+        assert surface.n_bonds == 2400
+
+        # 2D dimension provided
+        surface = Betacristobalite(dimensions=(5.3888 * 1.2, 4.589110 * 1.25))
+        assert surface.n_particles == 2850
+        assert surface.n_bonds == 3600
+
+        # 3D dimension provided
+        surface = Betacristobalite(
+            dimensions=(5.3888 * 1.2, 4.589110 * 1.25, 1.32 * 2)
+        )
+        assert surface.n_particles == 5700
+        assert surface.n_bonds == 7200
+
+        # Bogus values provided
+        with pytest.raises(ValueError):
+            surface = Betacristobalite(dimensions="bogus")
+
```

Comment:
Should add the test:
```suggestion
assert len(surface.available_ports()) > 1
distancesList = []
for particle in surface.particles():
    assert len(
        surface.particles_in_range(particle, dmax=0.001, max_particles=1)
        ) ==0
    
    
assert np.min(distancesList) > 0.001 # 1/1000 of a nm is too close           
```